System.Activities.Core.Presentation/System/Activities/Core/Presentation/SwitchDesigner.xaml

```diff
@@ -0,0 +1,195 @@
+﻿<? Copyright (c) Microsoft Corporation.  All rights reserved.?>
+<sad:ActivityDesigner x:Uid="sad:ActivityDesigner_1" x:Class="System.Activities.Core.Presentation.SwitchDesigner"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
```

Comment:
You seem to be missing the following from System.Activities.Core.Presentation\System\Activities\Core\Presentation

TransactionScopeDesigner.xaml
TransitionDesigner.xaml
TryCatchDesigner.xaml
VerticalConnector.xaml
WhileDesigner.xaml
WriteLineDesigner.xaml